### PR TITLE
Update changelog for v9.6.0

### PR DESCRIPTION
**Pull request by ryanml**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,177 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- [#10994](https://github.com/MetaMask/metamask-extension/pull/10994): Replacing the text "Seed Phrase" with "Secret Recovery Phrase" throughout the application.
+- [#10987](https://github.com/MetaMask/metamask-extension/pull/10987): Swaps: Improve hardware wallet UX
+- [#10936](https://github.com/MetaMask/metamask-extension/pull/10936): Swaps: Allow 0% slippage, show a warning for 0 < slippage <= 1, disallows Reviews when negative slippage occurs.
+- [#10946](https://github.com/MetaMask/metamask-extension/pull/10946): Send user to activity view after transaction is complete
+- [#10955](https://github.com/MetaMask/metamask-extension/pull/10955): Anchor the "Need Help" text to the bottom of the full page view
+- [#10767](https://github.com/MetaMask/metamask-extension/pull/10767): Improving error handling when gas prices are unavailable.
+- [#10836](https://github.com/MetaMask/metamask-extension/pull/10836): Improving "Review Swap" error handling
+- [#10800](https://github.com/MetaMask/metamask-extension/pull/10800): Swaps: Using lowercase value for the BSC contract address
```
Comment (danjm) [inline CHANGELOG.md:25]: `[#10800](https://github.com/MetaMask/metamask-extension/pull/10800): Swaps: Using lowercase value for the BSC contract address` should not be included in this changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,177 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Changed
+- [#11007](https://github.com/MetaMask/metamask-extension/pull/11007): Hide basic tab in advanced gas modal when on testnets.
+- [#10994](https://github.com/MetaMask/metamask-extension/pull/10994): Replacing the text "Seed Phrase" with "Secret Recovery Phrase" throughout the application.
+- [#10987](https://github.com/MetaMask/metamask-extension/pull/10987): Swaps: Improve hardware wallet UX
+- [#10936](https://github.com/MetaMask/metamask-extension/pull/10936): Swaps: Allow 0% slippage, show a warning for 0 < slippage <= 1, disallows Reviews when negative slippage occurs.
+- [#10946](https://github.com/MetaMask/metamask-extension/pull/10946): Send user to activity view after transaction is complete
+- [#10955](https://github.com/MetaMask/metamask-extension/pull/10955): Anchor the "Need Help" text to the bottom of the full page view
+- [#10767](https://github.com/MetaMask/metamask-extension/pull/10767): Improving error handling when gas prices are unavailable.
+- [#10836](https://github.com/MetaMask/metamask-extension/pull/10836): Improving "Review Swap" error handling
```
Comment (danjm) [inline CHANGELOG.md:24]: `[#10836](https://github.com/MetaMask/metamask-extension/pull/10836): Improving "Review Swap" error handling` should not be included in this changelog (was part of v9.5.0)

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,177 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- [#11025](https://github.com/MetaMask/metamask-extension/pull/11025): Fix redirection to the build quotes page from the swaps failed page
-- [#11011](https://github.com/MetaMask/metamask-extension/pull/11011): Update jsconfig excludes array
+### Changed
+- [#11007](https://github.com/MetaMask/metamask-extension/pull/11007): Hide basic tab in advanced gas modal when on testnets.
+- [#10994](https://github.com/MetaMask/metamask-extension/pull/10994): Replacing the text "Seed Phrase" with "Secret Recovery Phrase" throughout the application.
+- [#10987](https://github.com/MetaMask/metamask-extension/pull/10987): Swaps: Improve hardware wallet UX
+- [#10936](https://github.com/MetaMask/metamask-extension/pull/10936): Swaps: Allow 0% slippage, show a warning for 0 < slippage <= 1, disallows Reviews when negative slippage occurs.
+- [#10946](https://github.com/MetaMask/metamask-extension/pull/10946): Send user to activity view after transaction is complete
+- [#10955](https://github.com/MetaMask/metamask-extension/pull/10955): Anchor the "Need Help" text to the bottom of the full page view
```
Comment (danjm) [inline CHANGELOG.md:22]: ` [#10955](https://github.com/MetaMask/metamask-extension/pull/10955): Anchor the "Need Help" text to the bottom of the full page view` should not be included in this changelog, it was part of v9.5.0

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,177 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- [#10656](https://github.com/MetaMask/metamask-extension/pull/10656): Allow TextField to receive min and max attributes
-- [#10652](https://github.com/MetaMask/metamask-extension/pull/10652): Sync `master` with `develop`
-- Merge remote-tracking branch 'origin/develop' into sync-master
-- [#10614](https://github.com/MetaMask/metamask-extension/pull/10614): Move swaps constants to the shared constants directory
-- [#10640](https://github.com/MetaMask/metamask-extension/pull/10640): no more node:console
-- [#10634](https://github.com/MetaMask/metamask-extension/pull/10634): Setting balance to 0x0 when the original value is undefined
-- [#10615](https://github.com/MetaMask/metamask-extension/pull/10615): remove transactionCategory in favor of more types
-- [#10524](https://github.com/MetaMask/metamask-extension/pull/10524): Ensure permission log will only store JSON-able data
-- [#10624](https://github.com/MetaMask/metamask-extension/pull/10624): Replace logic for eth swap token in fetchQuotesAndSetQuoteState with getSwapsEthToken call
+- [#10963](https://github.com/MetaMask/metamask-extension/pull/10963): Ensure "Add Contact" button appears when a user already has contacts.
```
Comment (danjm) [inline CHANGELOG.md:35]: `[#10963](https://github.com/MetaMask/metamask-extension/pull/10963): Ensure "Add Contact" button appears when a user already has contacts.` should not be included, it was in v9.5.0

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,177 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- [#10952](https://github.com/MetaMask/metamask-extension/pull/10952): Prevent lag in external monitors when using Mac + Chrome
+- [#10884](https://github.com/MetaMask/metamask-extension/pull/10884): Prevent localStorage migration error in Firefox
```
Comment (danjm) [inline CHANGELOG.md:37]: `[#10884](https://github.com/MetaMask/metamask-extension/pull/10884): Prevent localStorage migration error in Firefox` should not be included, it was in v9.5.0

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,177 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- [#10774](https://github.com/MetaMask/metamask-extension/pull/10774): Swaps: Use network specific Swaps contract address upon checking approval
```
Comment (danjm) [inline CHANGELOG.md:38]: `[#10774](https://github.com/MetaMask/metamask-extension/pull/10774): Swaps: Use network specific Swaps contract address upon checking approval` should not be included, it was in v9.5.0

File: CHANGELOG.md [hunk 7]
```diff
@@ -7,177 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- [#10743](https://github.com/MetaMask/metamask-extension/pull/10743): Swaps: Use correct block explorer name and link in swaps when on custom network
```
Comment (danjm) [inline CHANGELOG.md:39]: All these issues with descriptions that begin `Swaps: ` should not be included, as they were in v9.5.0

Comment (ryanml) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 6 above)
Comment (ryanml) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 5 above)
Comment (ryanml) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 4 above)
Comment (ryanml) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (ryanml) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (ryanml) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 1 above)
Comment (ryanml) [reply CHANGELOG.md]: fixed

**Review by danjm — APPROVED** [review]

**Pull request merged**